packages/yoga/src/Checkbox/web/Checkbox.jsx

```diff
@@ -293,7 +295,7 @@ const Checkbox = ({
             checked={checked}
             disabled={disabled}
             {...(value ? { value } : {})}
-            {...rest}
+            {...restWithoutOnChange}
```

Comment:
Hello, normally the `onChange` should be tied to the Input and not the Wrapper... WDYT of inverting this?

Or better, explicitly extract the props that belong to a "input", and the "rest" pass it on the wrapper?